management_api_app/db/repositories/base.py

```diff
@@ -7,15 +8,21 @@
 class BaseRepository:
     def __init__(self, client: CosmosClient, container_name: str = None) -> None:
         self._client: CosmosClient = client
-        self._container: ContainerProxy = self.get_container(container_name)
+        self._container: ContainerProxy = self._get_container(container_name)
 
     @property
     def container(self) -> ContainerProxy:
         return self._container
 
-    def get_container(self, container_name) -> ContainerProxy:
+    def _get_container(self, container_name) -> ContainerProxy:
         try:
             database = self._client.get_database_client(config.STATE_STORE_DATABASE)
             return database.create_container_if_not_exists(id=container_name, partition_key=PartitionKey(path="/appId"))
         except Exception:
             raise UnableToAccessDatabase
+
+    def _query(self, query: str):
```

Comment:
_methods are not used here it seems. Are they for overriding in child class. Do they then need to be _ ?